pkg/apis/crd/v1alpha1/types.go

```diff
@@ -354,3 +355,129 @@ type BGPPeer struct {
 	// a restart before deleting stale routes. The range of the value is from 1 to 3600, and the default value is 120.
 	GracefulRestartTimeSeconds *int32 `json:"gracefulRestartTimeSeconds,omitempty"`
 }
+
+type PodReference struct {
+	Namespace string
+	Name      string
+}
+
+type ServiceReference struct {
+	Namespace string
+	Name      string
+}
+
```

Comment:
Do we need two definitions for these? I feel to define one "NamespacedNameReference" or "NameReference" is enough.

Reply:
ok, i prefer separated def too.